pkgs/build-support/writers/default.nix

```diff
@@ -150,10 +150,13 @@ rec {
       rustcArgs ? [],
       strip ? true
   }:
+  let
+    darwinArgs = lib.optionals stdenv.isDarwin [ "-L${lib.getLib libiconv}/lib" ];
```

Comment:
I don't understand this, isn't `lib.getLib` supposed to add the `/lib`? It doesn't for some reason.

Reply:
No, it isn't. It just gets you the `lib` output or, failing that, the `out` output.

Reply:
So should I remove it here since `libiconv` doesn't seem to have a `lib` output?

Reply:
No, the idea is that `getLib` does the correct thing independently of how the outputs are organized.